scss/components/_tooltips.scss

```diff
@@ -0,0 +1,77 @@
+.oo-tooltip::before {
+  content: attr(data-tip) ;
+  font-family: "Roboto Condensed", sans-serif;
+  text-shadow: none;
+  position:absolute;
+  z-index: 99999999;
```

Comment:
We should use a `$zindex` variable from _variables.scss here. We should also avoid using an arbitrarily large z-index since the customer might struggle to get their own popup menus, etc. on top of the player elements. I recommend testing with the value of `$zindex-popover`, if this is displayed correctly with ads then I think we could use that value.